deploy/charts/policy-approver/templates/namespace.yaml

```diff
@@ -0,0 +1,3 @@
+{{- if ne .Release.Namespace "cert-manager" }}
+{{- fail "namespace (--namespace, -n) must equal 'cert-manager'" }}
```

Comment:
So this was added because of the previous setup with having custom policy for the bootstapper.

Now that we have the tls provider strategy instead, this is not strictly necessary (but will need a flag to where to look for the policy-approver webhook CA Secret).

I'm in two minds, I do like having cert-manager being an enforced namespace, wdyt?

Reply:
I have made the namespace configurable.